frontend/simple/components/CreateGroup/CreateGroupMincome.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div>
+    <h1 class="title is-2 has-text-centered"><i18n>Minimum Income</i18n></h1>
+    <div class="field">
+      <div class="control">
+        <input
+          class="input is-large is-primary"
+          placeholder="Amount"
+          name="incomeProvided"
+          type="number"
+          step="1"
+          min="0"
+          required
+          :value="group.incomeProvided"
+          @input="update"
+          ref="mincome"
+        />
+      </div>
+    </div>
+    <p><i18n>How much income would you like your group to provide?</i18n></p>
```

Comment:
The mincome field should also have next to it a popup specifying the currency unit.